src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmethod/Example1.java

```diff
@@ -0,0 +1,41 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="JavadocMethod"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocmethod;
+
+// xdoc section -- start
+public class Example1 {
+
+  /**
+   *
+   */
+  Example1(int x) { // violation, param tag missing for x
+  }
+
+  /**
+   *
+   */
+  public int testMethod1(int p1) {  // violation, '@return tag should be present'
```

Comment:
`public int m1(int p1) { return p1; }`
Violation comment under method.

Apply same updates to other.

Reply:
Done